lib/train/transports/ssh.rb

```diff
@@ -150,7 +158,10 @@ def connection_options(opts)
         keys:                   opts[:key_files],
         password:               opts[:password],
         forward_agent:          opts[:forward_agent],
-        proxy_command:          opts[:proxy_command],
+        custom_proxy_command:   opts[:custom_proxy_command],
```

Comment:
I think this will break any current `proxy_command` usage. You could probably alias or set `custom_proxy_command` from `proxy_command` if set.

Reply:
@jquick I can't seem to think why I used a `custom` prefix then. Good suggestion. Fixed and pushed. Thanks for the suggestion.